Formula/mysql.rb

```diff
@@ -150,7 +150,7 @@ def caveats
       MySQL is configured to only allow connections from localhost by default
 
       To connect run:
-          mysql -uroot
+          mysql -u root
```

Comment:
Thanks for the PR. Are you sure this is a typo? Short flags (e.g. `-u`) typically accept an argument without requiring white space to separate them, so `-uroot` and `-u root` is likely the same.

Reply:
Both currently work, though [the MySQL docs](https://dev.mysql.com/doc/mysql-getting-started/en/#mysql-getting-started-connecting) do specify a separating space.